### PR TITLE
Release v20.2.0

### PR DESCRIPTION
**Pull request by rainchen**
# CHANGELOG

### v20.2.0

* change version naming style to `YY.MM.X` since CITA has changed it's version naming rule(https://github.com/citahub/cita/releases/tag/v20.2.0)
* add Makefile and update README, move changelog to CHANGELOG.md
* Bump rake from 10.5.0 to 12.3.3 fixes security vulnerability CVE-2020-8130
* migrate repo to citahub
* refactor codes with suggestion from make lint-ruby-code

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+			echo "No git tags found on current branch, please follow these steps:" ;\
+			echo "1. $$ git checkout master" ;\
+			echo "2. $$ git checkout -b update-changelog" ;\
+			echo "3. $$ git merge develop --no-edit" ;\
+			echo "4. $$ make changelog" ;\
+			echo "or just run $$ make changelog-auto" ;\
+			exit 1 ;\
+		fi
+
+changelog: changelog-check ## Generate CHANGELOG.md from git logs.
```
Comment (yangzhenlong) [inline Makefile]: his comment should go to the previous line

Comment (yangzhenlong) [inline Makefile]: this comment should go to the previous line

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+	$(info How do I make a good changelog? https://keepachangelog.com)
+	# auto install git-changelog
+	@git-changelog -v || pip3 install git-changelog
+	@OUTPUT=CHANGELOG.md ;\
+		git-changelog -s basic -t keepachangelog -o $${OUTPUT} . ;\
+		git diff $${OUTPUT} ;\
+		open $${OUTPUT} ;\
+		echo "Edit $${OUTPUT} to keep notable changes"
+
+changelog-auto: ## Auto generate CHANGELOG.md
```
Comment (yangzhenlong) [inline Makefile]: this comment should go to the previous line

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+	git checkout master
+	if git show-ref --verify --quiet "refs/heads/update-changelog"; then \
+		echo "Found update-changelog, auto delete it." ;\
+		git branch -D update-changelog ;\
+	fi
+	git checkout -b update-changelog
+	git merge develop --no-edit
+	make changelog
+
+make commit-release-notes: ## Commit lib/cita/version.rb and CHANGELOG.md
```
Comment (yangzhenlong) [inline Makefile]: this comment should go to the previous line

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+	@eval current_version=`$(get_current_version)` ;\
+	git add lib/cita/version.rb CHANGELOG.md ;\
+	git commit -m "bump version to v$${current_version}" ;\
+	git log -n 1
+
+current-version: ## Show current version number in lib/cita/version.rb file
```
Comment (yangzhenlong) [inline Makefile]: this comment should go to the previous line

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+	@$(call get_current_version)
+
+bump-version: ## Update version number in lib/cita/version.rb file, e.g.: bump-version v=1.2.0
```
Comment (yangzhenlong) [inline Makefile]: this comment should go to the previous line

File: Makefile (hunk 1 above)
Comment (rainchen) [reply Makefile]: this formatting is required by `help` receipt

Comment (yangzhenlong) [reply Makefile]: ok

**Comment by rink1969** [comment]
ci failed

**Pull request merged**